commons/src/main/scala/com/lynbrookrobotics/potassium/commons/drivetrain/Drive.scala

```diff
@@ -9,10 +9,18 @@ trait Drive {
   type DriveSignal
   type DriveVelocity
 
+  type DrivetrainHardware
+  type DrivetrainProperties
+
+  /**
+    * Drives with the signal with closed-loop control
+    * @param signal the signal to drive with
+    * @return
+    */
   protected def driveClosedLoop(signal: SignalLike[DriveSignal]): PeriodicSignal[DriveSignal]
```

Comment:
It's something that whoever extends this trait has to implement, only used internally and not as a proper controller.